src/types/position.ts

```diff
@@ -1,6 +1,11 @@
 import { Ref } from 'vue'
 import { Pixels } from '@/types/units'
 
+export type MousePosition = {
+  x: number,
+  y: number,
+}
+
```

Comment:
nit: I'd put this in useMousePosition.ts since its meant to be used with that. Its not really the same type of position as these other types.